### PR TITLE
Fix Arch build instructions again

### PR DESCRIPTION
**Pull request by drewkett**
npm is missing from the current instructions. Also, I added a note about the AUR as an alternative for building from source on Arch.  

This addresses issues raised by @Narrat in #7426 and @fusion809 in #7511.

### CODE REVIEW

File: docs/build-instructions/linux.md [hunk 1]
```diff
@@ -29,8 +29,10 @@ Ubuntu LTS 12.04 64-bit is the recommended platform.
 
 ### Arch
 
-* `sudo pacman -S gconf base-devel git nodejs libgnome-keyring python2`
-* `export PYTHON=/usr/bin/python2` before building Atom.
+* To automatically build Atom from source, install the atom-editor package from the [Arch User Repository (AUR)](https://wiki.archlinux.org/index.php/Arch_User_Repository)
```
Comment (mnquintana) [inline docs/build-instructions/linux.md]: We don't currently link to unofficial means of distributing Atom – would you mind taking out this line?

Comment (fusion809) [inline docs/build-instructions/linux.md]: I agree, the AUR is known to have its own set of issues, while the
atom-editor package is fairly stable from my experience, the AUR, in
general, is unreliable.

On 28 July 2015 at 11:07, Machisté N. Quintana <notifications@github.com>
wrote:

> In docs/build-instructions/linux.md
> <https://github.com/atom/atom/pull/8101#discussion_r35605982>:
>
> > @@ -29,8 +29,10 @@ Ubuntu LTS 12.04 64-bit is the recommended platform.
> >
> >  ### Arch
> >
> > -* `sudo pacman -S gconf base-devel git nodejs libgnome-keyring python2`
> > -* `export PYTHON=/usr/bin/python2` before building Atom.
> > +* To automatically build Atom from source, install the atom-editor package from the [Arch User Repository (AUR)](https://wiki.archlinux.org/index.php/Arch_User_Repository)
>
> We don't currently link to unofficial means of distributing Atom – would
> you mind taking out this line?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/atom/atom/pull/8101/files#r35605982>.
>



-- 
Thanks for your time,
Brenton

**Comment by drewkett** [comment]
Yea, that makes sense. I removed the mention of the AUR.

**Comment by kevinsawicki** [comment]
Thanks for fixing this :+1:

**Pull request merged**